### ISSUE TITLE
"

### ISSUE THREAD

**Issue by SanderDeWaal1992** [facebook/react#13038]
react-dom/cjs/react-dom-test-utils.development.js line 21 does require 'react-dom' but it does not have 'react-dom' in his dependency list in package.json.

line 21 of react-dom/cjs/react-dom-test-utils.development.js; "var ReactDOM = require('react-dom');"

Normally this would probably not result in problems, but our build environment does crash on this problem.

This problem is in react-dom@16.4.0.

**Comment by gaearon** [comment]
Sounds to me like this is a problem with your build environment. A package should be able to "require" itself if you're follow Node resolution mechanism (which you should for CommonJS bundles).

**Comment by SanderDeWaal1992** [comment]
@gaearon You are right, but as far as I know it should also be listed in the dependency list. Even if the dependency is the module itself, it is still a dependency.
